### ISSUE TITLE
Improve a11y test coverage for UI that requires user interaction to render or has an initial unmounted DOM state

### ISSUE THREAD

**Issue by m7kvqbe1** [Royal-Navy/design-system#2570]
See https://github.com/Royal-Navy/design-system/issues/2496 and https://github.com/Royal-Navy/design-system/issues/1928

UI that requires user interaction to render is currently not being covered by AXE a11y tests.

Investigate a way of improving coverage and in turn fix any a11y violations flagged as a result of this improved coverage.

**Comment by jpveooys** [comment]
Components still remaining:

- Masthead search
- Popover
